networking/decentralized-firewall/validator/validator.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
```

Comment:
Can you convert to using 2 spaces indent?

Reply:
Done.